python-libraries/nanover-core/src/nanover/trajectory/frame_data.py

```diff
@@ -38,6 +38,9 @@
 TOTAL_ENERGY = "energy.total"
 USER_ENERGY = "energy.user.total"
 
+SPARSE_USER_FORCES = "forces.user.sparse"
+SPARSE_USER_INDEX = "forces.user.index"
```

Comment:
let's match the names of fields more closely as:
`USER_FORCES_SPARSE`
`USER_FORCES_INDEX`

Reply:
All sorted